### DISCUSSION TITLE
Let's be fair Spritshards

### DISCUSSION THREAD

**Discussion by omgthisissoannoying** [chromiecraft/chromiecraft#5105] [Suggestions]
I can do dungeons all day for a week straight and I won't get any spirit shards, the vast majority are allaince so whenever the towers are horde in Terrokar, they will instantly be turned allaince. It's a cross server realm with one majority faction with a big M. So make it so the spirit shards drop for everyone. Pwetty pwease

**Comment by BeefSteaker** [comment]
Population seems to be about 60/40 from what I've seen. Plus once all the towers are captured they're locked down for 6 hours. 

Horde players just need to organise a group to take em

**Reply by omgthisissoannoying** [reply]
You're making those numbers up. And nobody wants to deal with this or someone obviously would have. All the let's call them super active players are allaince, they obviously can wait to take it over and over for a year, normies don't have time to jump on every 6th hour to wage war. You get from RL hop in online and just hope for the best. Here's what, tell me your problem with simply having it so both factions get shards, how would this hurt you why attempt to not let it happen.

**Reply by balleny** [reply]
seems more like you are making up an problem.
numbers are visible on the homepage (statistics) and getting shards as a horde player is less likely, but there have been plenty of groups organized a tower capturing
